src/main/java/org/terasology/launcher/ui/ApplicationController.java

```diff
@@ -492,21 +495,6 @@ protected void deleteAction() {
                 });
     }
 
-    /**
-     * Select the first item matching given predicate, select the first item otherwise.
-     *
-     * @param comboBox  the combo box to change the selection for
-     * @param predicate first item matching this predicate will be selected
-     */
-    private <T> void selectItem(final ComboBox<T> comboBox, Predicate<T> predicate) {
-        final T item = comboBox.getItems().stream()
-                .filter(predicate)
-                .findFirst()
-                .orElse(comboBox.getItems().get(0));
-
-        comboBox.getSelectionModel().select(item);
-    }
-
```

Comment:
is this unused, or why do we remove it?

Reply:
Yes, I wanted to commit with IntelliJ, and it said there is unused code in the file 🤷